Apps/WebClient/src/ClientApp/app/components/timeline/commentSection.vue

```diff
@@ -174,5 +202,53 @@ export default class CommentSectionComponent extends Vue {
         this.isLoadingComments = false;
       });
   }
+
+  private updateComment(): void {
+    this.isLoadingComments = true;
+    let commentPromise = this.commentService
+      .updateComment({
+        id: this.editing.id,
+        text: this.commentInput,
+        userProfileId: this.editing.userProfileId,
+        parentEntryId: this.editing.parentEntryId,
+        createdDateTime: this.editing.createdDateTime,
+        version: this.editing.version
+      })
+      .then(result => {
+        this.getComments();
+        this.commentInput = "";
+      })
+      .catch(err => {
+        console.log(err);
+        this.hasErrors = true;
+      })
+      .finally(() => {
+        this.isLoadingComments = false;
+        this.showInput = false;
+        this.isEditMode = false;
```

Comment:
show Input and isEditMode seem to be used for the same purposes

Reply:
The distinction being when a user submits a comment from an edit we need to know whether it's going to add the comment or update a comment, hence the isEditMode variable. Replacing it with showInput would always add comments since showInput is set to true when adding a comment too.